packages/core/addon/components/navi-loader.hbs

```diff
@@ -1,4 +1,4 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<div class="navi-loader__container" ...attributes>
+<div class="navi-loader__container {{@containerClass}}" ...attributes>
```

Comment:
Should we just update this line to pass a `class` instead of an arg: https://github.com/yavin-dev/framework/blob/044b09d877ad48d4f3cbf5a0cf9f4bf5505b961f/packages/reports/addon/templates/components/navi-loading-message.hbs